.github/workflows/build-test-publish.yml

```diff
@@ -134,6 +134,15 @@ jobs:
         run: dist/randovania/randovania --pytest --skip-gui-tests --skip-echo-tool --skip-resolver-tests --durations=100 --ignore=test/server --ignore=test/cli/commands/test_website.py --ignore=test/cli/commands/test_export_videos.py
         if: matrix.executable
 
+# TODO: this causes CI to hang
```

Comment:
Your choice between that and throwing this branch in the trash because I did not expect to get CI-only race conditions for this thing

Reply:
If we're not having it tested automatically in CI, I would like to have some other way of this being tested regularly that ideally doesn't involve "someone remembers to run this every once in a while".

Reply:
I first made this as a test that runs with pytest, but it was breaking weird things because this does shenanigans with the asyncio loop.

At least this will serve as a tool for profiling startup